ProcessLib/Utils/SetOrGetIntegrationPointData.h

```diff
@@ -0,0 +1,132 @@
+/**
+ * \file
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * Created on May 4, 2020, 10:27 AM
```

Comment:
This is not necessary - the information is stored in the commit.

Reply:
It records the moment of a mouse click to create the file (not the time at commit), and displays the time in documentation. If you strongly object it, I will remove it.